vaadin-testbench-core/src/main/java/com/vaadin/karibu/BaseKaribuTest.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (C) 2020 Vaadin Ltd
+ *
+ * This program is available under Commercial Vaadin Developer License
+ * 4.0 (CVDLv4).
+ *
+ *
+ * For the full License, see <https://vaadin.com/license/cvdl-4.0>.
+ */
+package com.vaadin.karibu;
```

Comment:
```suggestion
package com.vaadin.testbench.unit;
```
I think we should not have karibu in the package name as it should just be another way of tetsting with TestBench. 
Perhaps use `unit`, `junit` or `ui` to keep some clarity between class use cases.

Reply:
`com.vaadin.testbench.unit` was my first choice, then I changed it but only because `com.vaadin.karibu` was used in sample code.
I'll revert the packages to `com.vaadin.testbench.unit`